pkg/resourcemanager/iam/authorizers.go

```diff
@@ -260,13 +260,14 @@ func GetMSITokenForResource(resource string) (*adal.ServicePrincipalToken, error
 }
 
 // GetResourceManagementTokenHybrid retrieves auth token for hybrid environment
-func GetResourceManagementTokenHybrid(activeDirectoryEndpoint, tokenAudience string) (adal.OAuthTokenProvider, error) {
+// TODO(creds-refactor): this seems to be unused, remove it.
```

Comment:
Planning to do this in a follow-up PR, or was this intended to be part of this PR?

Reply:
Gah, I meant to do a pass across the PR to finalise these TODOs, sorry! May as well remove it here.